tests/openwisp2/settings.py

```diff
@@ -311,6 +316,37 @@
 
 OPENWISP_USERS_AUTH_API = True
 
+ASGI_APPLICATION = "openwisp2.routing.application"
+
+if TESTING:
+    DATABASES["default"]["TEST"] = {
+        "NAME": os.path.join(BASE_DIR, "test_openwisp_radius.db"),
+    }
```

Comment:
Can we adopt the same strategy as this (for consistency)?
https://github.com/openwisp/openwisp-network-topology/commit/f8bb68c60c16dab1ba0144e8a5d170a7f555b9e1#diff-26cea5c755fb048b49985aa06de44b90bfdfe6e9165f73f08bac2197df7e61e8R19-R22